tests/power_analysis/test_seed.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from cluster_experiments.perturbator import Perturbator
+from cluster_experiments.power_analysis import PowerAnalysis
+
+
+def get_config(perturbator: Perturbator) -> dict:
```

Comment:
```suggestion
def get_config(perturbator: str) -> dict:
```

I think it's a string

Reply:
True, changed it too